src/interpreter/executor/interpret.js

```diff
@@ -46,16 +47,15 @@ function setup(inputCode) {
     try {
         new Function(inputCode);
```

Comment:
Does this actually execute the user code, or just check that it's syntactically valid?

Reply:
This just checked syntax.  If we did `eval` it would execute but this is safe way to accomplish the same functionality.

Reply:
Okay cool. This won't get hung up on runtime errors, then.